backend/app/main/answer_views.py

```diff
@@ -66,7 +63,7 @@ def update_airtable_record(record_id, field_name, field_value):
         field_name: field_value
     }}
     requests.patch( 
-            "https://api.airtable.com/v0/appw4RRMDig1g2PFI/SMS%20Responses/{}".format(record_id),json=temp_field,
+            config[os.getenv("FLASK_CONFIG")].DATABASE_URL + "/{}".format(record_id),json=temp_field,
```

Comment:
Minor detail but you could change this to `"{}/{}".format(config[os.getenv("FLASK_CONFIG")].DATABASE_URL, record_id)` since you're already using the format function